lib/Dialect/FIRRTL/Transforms/Dedup.cpp

```diff
@@ -1002,7 +1011,7 @@ struct Deduper {
   // This maps an NLA to the operation or port that uses it. Since NLAs include
   // the name of the leaf element, its only possible for the NLA to be used by a
   // single op or port.
-  DenseMap<Attribute, AnnoTarget> targetMap;
+  DenseMap<Attribute, std::vector<AnnoTarget>> targetMap;
```

Comment:
I think this "hierarchical path user" map is likely what we want to add to NLATable to unify some of the needed functionality across all these passes.